documentation/Dumping.md

```diff
@@ -0,0 +1,18 @@
+# Dumping Games
+
+Dumping your games is the only legal way of obtaining commercial games, and the only allowed way to get games for Vita3K.
+
+However, if you're after homebrew games, you can search online or visit [VitaDB](https://vitadb.rinnegatamante.it/#/). Be warned that some homebrew apps may contain potentially unwanted software like malware.
+
+Please note that .pkg is for commercial games and .vpk is for homebrew games.
+
+# Dumping methods
+
+There are four ways to dump Vita games: MaiDump, [NoNpDrm](https://github.com/TheOfficialFloW/NoNpDrm), Vitamin and FAGdec. However, MaiDump and Vitamin are not supported by Vita3K. Please do not ask why Vitamin doesn't work.
```

Comment:
maidump is supported running in Vita3K, but the main is eboot.bin, many reasons are caused by eboot.bin